packages/theme/src/components/donut-chart.ts

```diff
@@ -0,0 +1,36 @@
+import { mergeMultiStyle, type ComponentMultiStyle } from "@yamada-ui/core"
+import { LineChart } from "./line-chart"
+
+export const DonutChart: ComponentMultiStyle = mergeMultiStyle(LineChart, {
+  baseStyle: {
+    donut: {},
+    activeShape: {},
+    inactiveShape: {},
+    label: { fillOpacity: 1 },
```

Comment:
Delete `fillOpacity` here and set it inside the component.

```tsx
  const labelClassName = useMemo(
    () => getClassName({ fillOpacity: 1, ...styles.label, ...label })(theme),
    [label, styles.label, theme],
  )
```

Reply:
Done.